app/keepers/modules.go

```diff
@@ -61,20 +70,18 @@ import (
 // AppModuleBasics returns ModuleBasics for the module BasicManager.
 var AppModuleBasics = []module.AppModuleBasic{
 	auth.AppModuleBasic{},
-	genutil.AppModuleBasic{},
+	genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
 	bank.AppModuleBasic{},
 	capability.AppModuleBasic{},
 	staking.AppModuleBasic{},
 	mint.AppModuleBasic{},
 	downtimemodule.AppModuleBasic{},
 	distr.AppModuleBasic{},
 	gov.NewAppModuleBasic(
-		append(
-			wasmclient.ProposalHandlers,
```

Comment:
were missing this + distrclient?

Reply:
WasmClient and DistributionClient no longer have them